Data-Structures/Stack/test/EvaluateExpression.test.js

```diff
@@ -0,0 +1,22 @@
+import { evaluateExpression } from '../EvaluateExpression.js';
+
+describe('evaluateExpression', () => {
+  it('should evaluate a valid expression', () => {
+    const expression = '3 4 * 2 / 5 +'; // (3 * 4) / 2 + 5 = 11
+    const result = evaluateExpression(expression);
```

Comment:
I would save myself this variable (I would probably save myself both variables, but the `expression` variable makes the comment with the expression in infix notation neater to write).

Reply:
Didn't quite catch this. Can you elaborate? Thank you!

Reply:
Since you only use the variables once, you could inline their expressions. So you could do `expect(evaluateExpression(expression)).toBe(11)`, or if you were to inline expression as well, `expect(evaluateExpression('3 4 * 2 / 5 +')).toBe(11)`, but then the comment `// (3 * 4) / 2 + 5 = 11` wouldn't be as close to the expression anymore.